contexts/logging-tls-context/README.md

```diff
@@ -0,0 +1,12 @@
+## Thread local based logging context for JVM
+
+This module provides the basic implementation of the logging context that
+uses thread local storage (TLS) to store the current context data.
+
+To use it, add `spine-logging-thread-local-context` to `runtimeOnly` configuration:
+
+```kotlin
+dependencies {
+    runtimeOnly("io.spine:spine-logging-thread-local-context:$version")
```

Comment:
Do we say "tls" or "thread local"?

Please recheck. And if there are two different names for the same thing (say, module name and artifact name), let's document it.